packages/EasyTest/src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyTest\Bridge\Symfony\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+final class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('easy_test');
+
+        $treeBuilder->getRootNode()
+            ->children()
+                ->booleanNode('enable_message_logger_listener_stub')
```

Comment:
Shouldn't it be `message_logger_listener_stub_enabled`? I don't remember whether we have a convention on that.

Reply:
<img width="727" alt="image" src="https://user-images.githubusercontent.com/13471770/215026179-50638a11-0ddb-4354-bf38-b17dc4fe1958.png">

Reply:
Please update other places in this PR correspondingly.

Reply:
Fixed